### ISSUE TITLE
Feature: auto-update, but only if TAGS already exist

### ISSUE THREAD

**Issue by Opioid** [jixiuf/ctags-update#3]
I believe it would be cool if ctags-update could be configured in a way that it will auto-update TAGS, but only if the file already exists.
The workflow would be as follows: I manually trigger TAGS generation in the directory of a project I want to use tags in and from that moment on the rest happens automatically. For all other projects in which I don't want to explicitly use tags in, nothing changes.

**Comment by jixiuf** [comment]
add option `ctags-update-prompt-create-tags`
```
(ctags-global-auto-update-mode)
(setq ctags-update-prompt-create-tags nil);you need manually create TAGS in your project
```

**Comment by Opioid** [comment]
EDIT2:
I see now that there is a delay of 5 minutes, that must have passed between tag updates to trigger it again on save. What is the rationale behind this? I guess updating the tags can become quite time intensive if the project grows beyond a certain size. Is that it? Is ctags smart enough to somehow re-use tags of files that have not been altered since last time?

EDIT1:
Ok, after updating the package I no longer get the error message.
However, saving a file will not lead to regenerating the tags. I tried it by adding something to a buffer, saving to file and check to see if the tag could be found afterwards. It could not be found.
I'm using etags-select, if that matters.

OUTDATED:

This gives me:

> Symbol's function definition is void: ctags-global-auto-update-mode

I tried something else instead, but it doesn't seem to work. (Or I explained poorly what I want.)
```
(use-package ctags-update)
(setq ctags-update-prompt-create-tags nil)
(add-hook 'js-mode-hook  'turn-on-ctags-auto-update-mode)
```
When I save a (js) file for which no tags exist, ctags will prompt me about creating tags. This is not what I want. When I save a file and (outdated) tags exist, they should be updated automatically. But if they don't exist, it should just be ignored.
